src/main/scala/com/peoplepattern/text/JaLangBundle.scala

```diff
@@ -0,0 +1,81 @@
+package com.peoplepattern.text
+
+import scala.collection.mutable.Buffer
+import scala.collection.JavaConverters._
+import com.atilika.kuromoji.ipadic.Tokenizer
+
+/**
+ * Custom language bundle for Japanese
+ *
+ * Uses Kuromoji for tokenization https://github.com/atilika/kuromoji
+ */
+object JaLangBundle extends LangBundle {
+
+  val stopwords = LangBundle.stopwords("ja")
+
+  /**
+   * Whether the char is in the ASCII range
+   */
+  private def asciichar(c: Char): Boolean = 0.toChar <= c && c <= 255.toChar
+
+  private def asciiSplit(text: String): Vector[(String, Boolean)] = {
+    var tmp = text
```

Comment:
I don't love the String-valued tmp, and the amount of string stuff like this below:
```scala
tmp = tmp.drop(nonascii.size)
```